lib/fullstory/index.js

```diff
@@ -98,6 +105,8 @@ function convert (trait, value) {
 
 /**
  * Check if n is a float.
+ *
+ * @param {Number} n
```

Comment:
Seems like a noisy addition. If it's not part of our style guide, let's remove it!